lib/fund.js

```diff
@@ -131,8 +122,10 @@ async function openFundingUrl ({ path, tree, spec, fundingSourceNumber }) {
   if (!funding) {
```

Comment:
Could this whole section be written to use a `const`?  Maybe something like:

```js
const { funding } = retrievePackageMetadata() || 
  await (pacote.manifest(arg, npm.flatOptions).catch(() => ({}))
```

Reply:
that looks neat! yep, will change it 😊